packages/lib-classifier/src/plugins/tasks/SurveyTask/components/components/Choice/helpers/checkFilledIn.js

```diff
@@ -0,0 +1,29 @@
+import getQuestionIds from './getQuestionIds'
+
+export default function checkFilledIn (
```

Comment:
this is trivial, but I'm not wild about this function name. maybe `allowIdentify` or something else?

Reply:
Yeah, I like that or maybe `allowIdentification` implies the return value is a boolean